git_machete/cli.py

```diff
@@ -75,274 +77,423 @@ def usage(command: str = None) -> None:
+    anno_parser.add_argument('annotation_text', nargs='*')
+    anno_parser.add_argument('-b', '--branch', default=argparse.SUPPRESS)
+    anno_parser.add_argument(
+        '-H', '--sync-github-prs', action='store_true', default=argparse.SUPPRESS)
+    anno_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    delete_unmanaged_parser = subparsers.add_parser('delete-unmanaged', add_help=False)
+    delete_unmanaged_parser.add_argument(
+        '-y', '--yes', action='store_true', default=argparse.SUPPRESS)
+    delete_unmanaged_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    diff_full_parser = subparsers.add_parser('diff', aliases=['d'], add_help=False)
+    diff_full_parser.add_argument('branch', nargs='?')
+    diff_full_parser.add_argument(
+        '-s', '--stat', action='store_true', default=argparse.SUPPRESS)
+    diff_full_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    discover_parser = subparsers.add_parser(
+        'discover', argument_default=argparse.SUPPRESS, add_help=False)
+    discover_parser.add_argument('-C', '--check-out-since')
+    discover_parser.add_argument('-l', '--list-commits', action='store_true')
+    discover_parser.add_argument('-r', '--roots')
+    discover_parser.add_argument('-y', '--yes', action='store_true')
+    discover_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    edit_parser = subparsers.add_parser('edit', aliases=['e'], add_help=False)
+    edit_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    file_parser = subparsers.add_parser('file', add_help=False)
+    file_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    fork_point_parser = subparsers.add_parser(
+        'fork-point', argument_default=argparse.SUPPRESS, add_help=False)
+    fork_point_parser.add_argument('branch', nargs='?')
+    fork_point_exclusive_optional_args = fork_point_parser.add_mutually_exclusive_group()
+    fork_point_exclusive_optional_args.add_argument('--inferred', action='store_true')
+    fork_point_exclusive_optional_args.add_argument('--override-to')
+    fork_point_exclusive_optional_args.add_argument('--override-to-inferred', action='store_true')
+    fork_point_exclusive_optional_args.add_argument('--override-to-parent', action='store_true')
+    fork_point_exclusive_optional_args.add_argument('--unset-override', action='store_true')
+    fork_point_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    format_parser = subparsers.add_parser('format', add_help=False)
+    format_parser.add_argument('-h', '--help', action=MacheteHelpAction)
+
+    github_parser = subparsers.add_parser(
+        'github', argument_default=argparse.SUPPRESS, add_help=False)
+    github_parser.add_argument(
+        'subcommand', choices=['anno-prs', 'checkout-prs', 'create-pr', 'retarget-pr'])
+    github_parser.add_argument('pr_no', nargs='?')
+    github_parser.add_argument('--draft', action='store_true')
```

Comment:
Nit: `git machete github anno-prs --draft` now passes... it theoretically shouldn't, `--draft` only makes sense for `create-pr`